pkgs/top-level/go-packages.nix

```diff
@@ -2671,6 +2665,9 @@ let
       pushover
     ];
 
+    # Tests exist, but seem to clash with the firewall.
+    doCheck = false;
+
     buildFlagsArray = ''
       -ldflags=
           -X main.buildVersion=${rev}
```

Comment:
It looks like the metadata ldflags used in the alertmanager build have changed in the rewrite: https://github.com/prometheus/alertmanager/blob/master/scripts/build.sh#L31

Those are just used on the status page I believe, but it's nice to have some useful data in there.